### ISSUE TITLE
Review fact sheet to assess if any info is missing from our HLR page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#34816]
## Issue Description
Decision reviews stakeholders asked us to link to a fact sheet PDF from our Higher-Level Review page. Since we are trying to get away from linking to PDFs for accessibility reasons, we need to see if there is information that should be added to our benefit detail page. 

We need to review the document and compare the information to what's currently on our [Higher-Level Review page](https://www.va.gov/decision-reviews/higher-level-review/). We should note anything missing from our page in this ticket for Beth.

Fact sheet: [https://app.zenhub.com/files/133843125/d774d606-a14d-4fbd-b02f-91b38b00da72/download](https://app.zenhub.com/files/133843125/d774d606-a14d-4fbd-b02f-91b38b00da72/download)

## Tasks
- [x] Review fact sheet (Selina, Patrick, Randi)
- [x] List out what's missing from our tier 1 page (Selina, Patrick, Randi)

## Acceptance Criteria
- [x] All missing information is listed for Beth to review

**Comment by lunascoop** [comment]
Here are some suggestions:

**Page intro**
- Add in the fact that their representative can request an HLR: If you disagree with VA’s decision, you or your representative can request to have a senior reviewer take a new look at your case. 

**Can I request a Higher-Level Review? section** 
- The question title seems a bit redundant, and on the [Supplemental Claims](https://www.va.gov/decision-reviews/supplemental-claim/) and [Board Appeals](https://www.va.gov/decision-reviews/board-appeal/) pages too. The page intro already tells Veterans that they can request an HLR. Perhaps rephrase to say: **When can I request a Higher-Level Review?** and combine the first two sentences: "You have one year from the date on your decision letter to request a Higher-Level Review of an initial claim or Supplemental Claim decision."
- For 508 compliance, drop link to contested claim to another line: Learn more about contested claims

**For Note:**

- Perhaps make it an H2 (What should I know before requesting a Higher-Level Review?). This same note on the [Fiduciary Claims](https://www.va.gov/decision-reviews/fiduciary-claims/) page is part of an H2.
- For the second sentence of the second bullet, be more specific as to where the Veteran can choose the optional call for an HLR: "If you want to speak with a reviewer on the phone, you'll need to mark the circle in item 16A when filling out VA Form 20-0996, Decision Review Request: Higher-Level Review."
- Add this note: Only one optional call (informal conference) is allowed for each HLR. Make sure that you will have enough time to talk to the higher level reviewer to explain any errors of fact or law in your case.

**For the What happens next? section**
- It's a bit odd that it starts with a link to another page. Can we end the section w/that link instead?

**Comment by bethpottsVADEPO** [comment]
@lunascoop @RLHecht @psanders86 Thanks for this consideration and write-up. Since we don't have the final fact sheet yet to work from anyway, I'm going to push my assessment of this to Monday. Will post here after I review.

**Comment by bethpottsVADEPO** [comment]
@RLHecht @lunascoop @psanders86 Thanks for thinking this through and for the initial approach you outlined here! I bolded my feedback below. I think it makes sense to get these changes into drupal so it's queued up when we receive v2 of the fact sheet. I doubt there will be much else (if anything) to do on our page based their iteration of the fact sheet.

Page intro: 
**Yes, thank you for adding in the mention of the rep**

Add in the fact that their representative can request an HLR: If you disagree with VA’s decision, you or your representative can request to have a senior reviewer take a new look at your case.

For Note: 
**1/ The guidance about selecting the radio button in 16A may be specific to the PDF application. I'm not sure if it applies to the online app as well. You'll need to use a test user account to verify that it's in the online app and how it appears there (probably won't have a 16A at all, but not sure). Then figure out how to phrase this action of selecting the informal conference for both use cases.
2/ For the sentence about only one optional call allowed, make that sentence active voice.** 

For the second bullet, be more specific as to where the Veteran can choose the optional call for an HLR: "If you want to speak with a reviewer on the phone, you'll need to mark the circle in item 16A when filling out VA Form 20-0996, Decision Review Request: Higher-Level Review." Also be more specific in the first sentence than speaking with a reviewer on the phone is considered an informal conference.
Add this note: Only one optional call (informal conference) is allowed for each Higher-Level Review. Make sure that you have enough time to speak with the higher level reviewer to explain any errors for your case.


Initial suggestions for future changes to the HLR page

For Note: **Yes, please add an H2!**

Perhaps make it an H2 (What should I know before requesting a Higher-Level Review?). This same note on the Fiduciary Claims page is part of an H2.

**Comment by lunascoop** [comment]
@RLHecht I've added the changes in Drupal. Let me know if we need to sync on this, especially the change to the second bullet (adding guidance on how to request an informal conference when applying for an HLR).

Higher-Level Reviews: http://preview-prod.vfs.va.gov/preview?nodeId=3003

**Comment by RLHecht** [comment]
Just looked this over @lunascoop. I reworked the guidance if you're applying by mail or in person to include the VA form name. Let me know what you think.

**Comment by RLHecht** [comment]
Moving this to blocked until we get the new fact sheet from stakeholders.

**Comment by RLHecht** [comment]
Checked in with stakeholders on 2.23 and the fact sheet hasn't been approved yet.

**Comment by RLHecht** [comment]
@lunascoop Here's the finalized fact sheet back from the stakeholders. Could you please compare it to the changes we made on the HLR page and make any new edits if they are needed? Thanks!

[https://app.zenhub.com/files/133843125/74c05951-586b-4006-8e42-cd3ab823a421/download](https://app.zenhub.com/files/133843125/74c05951-586b-4006-8e42-cd3ab823a421/download)

**Comment by lunascoop** [comment]
@psanders86 @RLHecht I've reviewed the updated PDF and updated the HLR page with additional content. Specifically, I added the content about an HLR taking longer if an informal conference is requested. I added this to the number callout section as a note since the content for the number callout is about the time it takes to make a decision. I could also be added to the Note section as a bullet point. Let me know your thoughts. 

Higher-Level Reviews: http://preview-prod.vfs.va.gov/preview?nodeId=3003

**Comment by RLHecht** [comment]
@bethpottsVADEPO The updated Drupal preview is ready for you: http://preview-prod.vfs.va.gov/preview?nodeId=3003

**Comment by bethpottsVADEPO** [comment]
@RLHecht Just reviewed and edited the new content. Take a look at what I did, then you can communicate with stakeholders about it. Thanks!

**Comment by RLHecht** [comment]
The stakeholders approved our updates and I've published the page. I'll let them know once it's live.
